pkg/cmd/plugincheck/main.go

```diff
@@ -13,7 +13,29 @@ import (
 	"github.com/grafana/plugin-validator/pkg/plugin"
 )
 
+// Deprecated: plugincheck V1 is deprecated and will be removed in a future release.
+// Use plugincheck2 instead. See https://github.com/grafana/plugin-validator
 func main() {
+
+	fmt.Println(`
+     888                                               888                888
+     888                                               888                888
+     888                                               888                888
+ .d88888 .d88b. 88888b. 888d888 .d88b.  .d8888b 8888b. 888888 .d88b.  .d88888
+d88" 888d8P  Y8b888 "88b888P"  d8P  Y8bd88P"       "88b888   d8P  Y8bd88" 888
+888  88888888888888  888888    88888888888     .d888888888   88888888888  888
+Y88b 888Y8b.    888 d88P888    Y8b.    Y88b.   888  888Y88b. Y8b.    Y88b 888
+ "Y88888 "Y8888 88888P" 888     "Y8888  "Y8888P"Y888888 "Y888 "Y8888  "Y88888
+                888
+                888
+                888
+
+	**plugincheck V1 is deprecated and you should not use it.
```

Comment:
Nit: do we need a closing `**` as well to make it bold? (In case that was the intention)

Reply:
Not really this is not supposed to be markdown, just a way to catch attention in the terminal